### DISCUSSION TITLE
QoL wrapper for vec of wallets

### DISCUSSION THREAD

**Discussion by MujkicA** [FuelLabs/fuels-rs#634] [Ideas]
In a recent PR, @iqdecay suggested we implement some methods on `Vec<Wallets>` to provide eg the vec of their addresses, the vec of their balances etc.

**Comment by MujkicA** [comment]
IMO this seems like a good QoL improvement. The only downside I can think of is that the user has to be introduced to yet another abstraction before they can finally work with wallets.

**Reply by iqdecay** [reply]
Is it really a another abstraction layer though? In the sense that it would be transparent for the user from a usage perspective:
`wallets.get_balances` vs `wallet.get_balance`. To be clear I'm suggesting this as an addition, **not** a replacement for current existing methods. Obviously the user wouldn't have to use a single-element `Vec` of a wallet to interact with it.
